docs/concepts/governance/maintainers.md

```diff
@@ -0,0 +1,18 @@
+# Maintainers
```

Comment:
I'm not sure we should be burying these files.

Reply:
I'm +1 on bumping it for governance. In my mind, it's a "special" concept that we want to give big priority.

On a different note, it'd be great if we could link/pull these directly from the governance repo.

Reply:
Oh, good point @kevingoldsmith, I'll move them back and use a symlink to have the real files in the root of the repo, but also render them on the docs site (we can make a editorial decision if we want to keep them separately).

@fpliger FWIW this structure was recommended by the [MVG](https://github.com/github/MVG), to have some files in the project repo, and some on the org level. So if we were to add additional projects to the pyscript organization, they would require also at least a `MAINTAINERS.md` and `GOVERNANCE.md`.